net-irc/kvirc/kvirc-5.2.0.ebuild

```
@@ -0,0 +1,157 @@
+# Copyright 2009-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+DESCRIPTION="Advanced IRC Client"
+HOMEPAGE="http://www.kvirc.net/ https://github.com/kvirc/KVIrc"
```

```suggestion
HOMEPAGE="https://www.kvirc.net/ https://github.com/kvirc/KVIrc"
```
also for the `9999` ebuild.

---

Yes, the kvirc website got finally fixed after I opened this PR.

Updated.